HTML - 38/memorizeTheOrderOfRedCards.html

```diff
@@ -0,0 +1,60 @@
+        <th id="13" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="14" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+    </tr>
+    <tr>
+        <td id="15" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></td>
+        <th id="16" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="17" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="18" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="19" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+    </tr>
+    <tr>
+        <td id="20" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></td>
+        <th id="21" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="22" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="23" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="24" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+    </tr>
+    <tr>
+        <td id="25" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></td>
+        <th id="26" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="27" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="28" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+        <th id="29" onclick="trying(this.id)" style="border: 1px solid black; border-spacing: 5px; width: 100px; height: 100px; background-color: #267326"></th>
+    </tr>
+</table>
+<br>
+<form>

```

Comment:
When you are using form instead of adding onclick to button you should use submit input and add onsubmit to form.

Reply:
You mean to make the button a submit button? Will try it